src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputTryCatchIfElse.java

```diff
@@ -34,15 +36,21 @@ void foo() throws Exception {
     try (MyResource r = new MyResource()) {}
 
     try (MyResource r = new MyResource()) {} catch (Exception expected) {}
+    // violation above ''}' at column 74 should be alone on a line.'
 
     try (MyResource r = new MyResource()) {} catch (Exception expected) { }
-    // violation above 'Empty blocks should have no spaces.'
+    // 2 violations above:
+    //  'Empty blocks should have no spaces.'
+    //  ''}' at column 75 should be alone on a line.'
 
     try (MyResource r = new MyResource()) {
+      // violation below ''}' at column 35 should be alone on a line.'
     } catch (Exception expected) {}
 
     try (MyResource r = new MyResource()) {
-      // violation below 'Empty blocks should have no spaces.'
+      // 2 violations 3 lines below:
+      //  'Empty blocks should have no spaces.'
+      //  ''}' at column 36 should be alone on a line.'
     } catch (Exception expected) { }
```

Comment:
test is failing for `''}' at column X should be alone on a line.'`, suppression isn't able to detect it. It is being false-negative

Reply:
test result:

```
Violation lines for C:\OpenSource\checkstyle\src\it\resources\com\google\checkstyle\test\chapter4formatting\rule412nonemptyblocks\InputTryCatchIfElse.java differ.
missing (4): 38, 41, 48, 54
```

Reply:
please fix CI, and diff in code will show that we loosing good violation.